Worldbuilding/Background/Creating New Species.md

```diff
@@ -1,53 +1,74 @@
+
+### Natives of the [[Outer Realms]]
+
+9. **Children of Creation**. Any creature or species that arises directly from [[~soulstuff~]] and is native to the [[Outer Realms]] fits in this category. Classic examples include fiends and celestials. A fundamental aspect of these creatures and species is that they cannot be killed on the material plane: if they are defeated in the [[Inner Realms]], they simply dissipate and reform on their native plane.
+10. **Divinities**. Any creature that is a pure representation of the energy of the [[Plane of Creation]] can be considered a divinity. Divinities -- which include the [[Embodied Gods]] and the [[Incorporeal Gods]], as well as creatures such as the [[Tanshi]] and the [[Kestavo]] -- are distinct from the Children of Creation primarily in terms of their power. Left undetermined is whether powerful archdevils and demon lords and the like are more akin to Divinities or Children of Creation. 
+
+### Natives of the Far Realms
 
-If yes, and it is supposed to be an “evil” race, you probably want 7 if it is clearly native to Taelgar, or 5 if it is supposed to be alien and unusual. Although, if it has strong connections to fey, giants, dragons, or elements, 1 is also plausible.
+11. **Alien Species**. There are many things flung across the universe that are older, and stranger, than the world of Taelgar. Many aberrations, and especially mind flayers, are ultimately invaders from these Far Realms, and many of these invaders created twisted creations to be their slaves, prey, or entertainment, some of whom still linger in the world. While at the moment, the only canonical invaders to Taelgar are from ancient and disturbing Far Realms, some races and creatures that are traditionally associated with Spelljammer in D&D could have an origin via this mechanism, or via a similar “transport from other timelines” type mechanism. 
 
-If yes, and it is *not* supposed to be an evil race, consider whether it could have been related to fey, giants, dragons, or elementals, in which case 1. Otherwise, consider 9.
+## Decision Tree for Adding New Races
+
+Is the creature part of a civilization with a history, culture, religious beliefs, language, diversity of thought and opinion? 
+
+- No: You are likely looking 7 or 8, or maybe 5; the choice depends largely on lore and fit. In general, 7 is more low-intelligence “natural” species, 8 is more high intelligence “cursed” species, and 5 is more "corrupted" species, but YMMV. For example, hydra are likely 7; dopplegangers are likely 8; medusa are likely 5. 
```

Comment:
I wouldn't say don't merge this, but I think the distinction between 3 (dragons/fey/giants/elements), 4 ("awakened") and 6 ("other") is probably not that relevant, or at least not that well captured by this document.

We can talk more about this and it doesn't have to hold up a merge, but I was liking the idea that within the "inner realms" category, the only really clear canonical division is between 1 (embodied), 2 (human), other (3/4/5/6)

I think this as background encourages a proliferation of intelligent species as long as they can be fit into the "dragons/fey/giants/elements" frame which I am not sure is correct. My take would be basically a new species should either be:

(a) category 1 (humans) / 2 (firstborn) / 5 (corruptions) / 7 (beasts) / 8 (monsters/magic) / 9 (outer planes) / 10 (gods) / 11 (far realm)
(b) considered how it could be related to those
(c) thought about carefully

The current canonical species not in 1/2/5/7/8/9/10/11 is, I think
kenku
giants
kobolds
merfolk
centaurs

plus "fey" broadly defined and "elementals" broadly defined
plus whatever dragons are

Does that list really have to be designed to be able to accommodate all kinds of other ideas? I mean, if one of us decides introducing Yuan-tan or Thrikeen or Liontaurs is really a cool idea, fine, but they probably need a unique story and a reason to exist. I'm not sure that there needs to be an unbounded number of complex races (and villians can always be void mind corruptions - category 5 - which provides for a good reason for various "bad guys" to fight)